CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The public API of this library consists of the functions declared in file
 [h3api.h.in](./src/h3lib/include/h3api.h.in).
 
 ## [Unreleased]
+### Added
+- CMake options for excluding filter applications or benchmarks from the build.
```

Comment:
Nit: Add PR # ?

As an aside, if we required PR numbers for all lines in the Changelog, it might make scripting the release check more reliable.

Reply:
The trouble is I would need to know the PR number before creating the PR.